ci/deploy_tyk_pro.sh

```diff
@@ -5,34 +5,41 @@ set -e
 NAMESPACE=tykpro-control-plane
 PRODIR=${PWD}/ci/tyk-pro
 ADMINSECRET="54321"
+TIMEOUT=60s
 
-echo "creating namespace ${NAMESPACE}"
-kubectl create namespace ${NAMESPACE}
+
+
+if OUTPUT=$(kubectl get namespaces 2> /dev/null | grep "${NAMESPACE}") ; then
+   echo "namespace ${NAMESPACE} already exists"
+else
+  echo "creating namespace ${NAMESPACE}"
+  kubectl create namespace ${NAMESPACE}
+fi
 
 echo "deploying databases"
 kubectl apply -f "${PRODIR}/mongo/mongo.yaml" -n ${NAMESPACE}
 kubectl apply -f "${PRODIR}/redis" -n ${NAMESPACE}
 
 echo "waiting for redis"
-kubectl wait deployment/redis -n ${NAMESPACE} --for condition=available --timeout=60s
+kubectl wait deployment/redis -n ${NAMESPACE} --for condition=available --timeout=${TIMEOUT}
 echo "waiting for mongo"
-kubectl wait deployment/mongo -n ${NAMESPACE} --for condition=available --timeout=60s
+kubectl wait deployment/mongo -n ${NAMESPACE} --for condition=available --timeout=${TIMEOUT}
 
 echo "creating configmaps"
-kubectl create configmap -n ${NAMESPACE} dash-conf --from-file "${PRODIR}/dashboard/confs/dash.json"
-kubectl create configmap -n ${NAMESPACE} tyk-conf --from-file "${PRODIR}/gateway/confs/tyk.json"
+kubectl create configmap -n ${NAMESPACE} dash-conf --dry-run --from-file "${PRODIR}/dashboard/confs/dash.json" -o yaml | kubectl apply -f -
+kubectl create configmap -n ${NAMESPACE} tyk-conf --dry-run --from-file "${PRODIR}/gateway/confs/tyk.json" -o yaml | kubectl apply -f -
 
 echo "setting dashboard secrets"
-kubectl create secret -n ${NAMESPACE} generic dashboard --from-literal "license=${TYK_DB_LICENSEKEY}" --from-literal "adminSecret=${ADMINSECRET}"
+kubectl create secret -n ${NAMESPACE} generic dashboard --dry-run --from-literal "license=${TYK_DB_LICENSEKEY}" --from-literal "adminSecret=${ADMINSECRET}" -o yaml | kubectl apply -f -
 kubectl get secret/dashboard -n tykpro-control-plane -o json | jq '.data.license'
```

Comment:
`jq` features and syntax vary widely based on version. Since kubectl supports JSONPath, you can drop the dependency on jq thusly,
```
kubectl get secret/dashboard -n ${NAMESPACE} -o jsonpath='{.data.license}'
```

Reply:
changed